packages/core/src/payment/strategies/braintree/braintree.ts

```diff
@@ -6,7 +6,7 @@ import {
     TokenizePayload,
 } from '../googlepay';
 import { PaypalAuthorizeData, PaypalButtonOptions, PaypalButtonRender, PaypalSDK } from '../paypal';
-
+import { Omit } from '../../../common/types';
```

Comment:
Omit is a default method for typescript. This line can be removed.

Reply:
I see, but I think in our repo this is required, otherwise, a build error will be thrown. I can see quite a few places are introducing it.

Reply:
This was added a long time ago before our version of TS had the method, can you please try the native `Omit` and see if it works fine.

Reply:
I tried, but it is not working. 😢 That is the reason I added it here